src/overlays/actors/ovl_Bg_Dblue_Movebg/z_bg_dblue_movebg.h

```diff
@@ -7,11 +7,50 @@ struct BgDblueMovebg;
 
 typedef void (*BgDblueMovebgActionFunc)(struct BgDblueMovebg*, GlobalContext*);
 
+#define BGDBLUEMOVEBG_GET_F(thisx) ((thisx)->params & 0xF)
+#define BGDBLUEMOVEBG_GET_FF0(thisx) (((thisx)->params >> 4) & 0xFF)
+#define BGDBLUEMOVEBG_GET_F000(thisx) (((thisx)->params >> 0xC) & 0xF)
+
 typedef struct BgDblueMovebg {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x18];
+    /* 0x0000 */ DynaPolyActor dyna;
     /* 0x015C */ BgDblueMovebgActionFunc actionFunc;
-    /* 0x0160 */ char unk_160[0x210];
+    /* 0x0160 */ s32 unk_160;
+    /* 0x0164 */ Gfx* unk_164;
+    /* 0x0168 */ s32 unk_168;
```

Comment:
This is a pointer to a display list
```suggestion
    /* 0x0168 */ Gfx* unk_168;
```